osu.Desktop/Program.cs

```diff
@@ -186,7 +191,11 @@ private static void setupVelopack()
         [SupportedOSPlatform("windows")]
         private static void configureWindows(VelopackApp app)
         {
-            app.WithFirstRun(_ => WindowsAssociationManager.InstallAssociations());
+            app.WithFirstRun(_ =>
+            {
+                isFirstRun = true;
+                WindowsAssociationManager.InstallAssociations();
+            });
```

Comment:
I believe you'll want to put this in the parent method, because as-is this will only run on Windows.

i.e. I think it should be fine to:
```
app.WithFirstRun(_ => isFirstRun = true);
```
```

Reply:
Oh, good catch.

Reply:
I've tested on macOS and it doesn't seem that this code will run on re-"install". Which makes sense since there's no installer.

I guess it is what it is for now and the user will have to switch their release stream.